api/types/access_request.go

```diff
@@ -776,6 +781,28 @@ func (f *AccessRequestFilter) FromMap(m map[string]string) error {
 
 // Match checks if a given access request matches this filter.
 func (f *AccessRequestFilter) Match(req AccessRequest) bool {
+	// only return if the request was made by the api requester
```

Comment:
my initial gut reaction is that how do we ensure that the filter request is actually the context user?

Reply:
the only place its set (and overwritten if incorrectly set by some client) is in that one spot. if for a specific user that isn't the person making the api call, thats the `req.Filter.User` field

Reply:
gotcha, i looked at the api definition of `requester` and one suggestion is to include in the api comment that it's for `internal use only`.